### PR TITLE
Project Extent | Usability enhancements for identifying streets vs urban trails

### PR DESCRIPTION
**Pull request by mddilley**
Closes https://github.com/cityofaustin/atd-data-tech/issues/5781

This PR updates the streets layer styles to consider the "off-street" property when applying color to the layer. The streets are still the primary blue color, but the trails ("off-street") are now shown in green to help differentiate them. I also updated the `useHoverLayer` hook to get a text string from features to show in a tooltip on hover. For the streets layer, the tooltip now shows the street name if it is available from the vector tile layer.

### Updated colors for the street layer showing trails in green at Mopac & Davis
Screen Shot 2021-04-20 at 2 34 57 PM

### Street names on hover
streetHover

### CODE REVIEW

**Comment by amenity** [comment]
Wow!! Looks so good. Thanks for sneaking in these street names, too. 🙏

Everything works as expected seeing+selecting trails here: https://deploy-preview-282--atd-moped-main.netlify.app/moped/projects/176

Creating a new project, I got hung up on `Error: invalid input syntax for type numeric: ""` which I believe means there is a database update that can't be reflected on Netlify?
Screen Shot 2021-04-20 at 5 43 01 PM

File: moped-editor/src/utils/mapHelpers.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ import { get } from "lodash";
 
 export const MAPBOX_TOKEN = process.env.REACT_APP_MAPBOX_TOKEN;
 export const drawnLayerName = "drawnByUser";
+const TRAIL_LINE_TYPE = "Off-Street";
```
Comment (sergiogcx) [inline moped-editor/src/utils/mapHelpers.js:10]: It shouldn't go without saying, I like the color and it makes it easy to the eye and it makes the app look more Kawaii

File: moped-editor/src/theme/index.js [hunk 2]
```diff
@@ -23,6 +23,7 @@ const theme = createMuiTheme({
     },
     map: {
       transparent: "rgba(0,0,0,0)",
+      trail: "#4caf50",
```
Comment (sergiogcx) [inline moped-editor/src/theme/index.js:26]: It shouldn't go without saying, I like the color and it makes it easy to the eye and it makes the app look more Kawaii

File: moped-editor/src/utils/mapHelpers.js [hunk 3]
```diff
@@ -232,6 +239,15 @@ export const getLayerNames = () => Object.keys(mapConfig.layerConfigs);
...
 
+/**
+ * Get a feature's property that contains text to show in a tooltip
+ * @param {object} feature - GeoJSON feature taken from a Mapbox click or hover event
+ * @param {string} layerName - Name of layer to find tooltip text property from layer config
+ * @return {string} The text to show in the tooltip
+ */
+export const getFeatureHoverText = (feature, layerName) =>
+  feature.properties[mapConfig.layerConfigs[layerName].tooltipTextProperty];
+
```
Comment (sergiogcx) [inline moped-editor/src/utils/mapHelpers.js:250]: I noticed this! Way to go! 🙌

**Review by sergiogcx — APPROVED** [review]
The map keeps looking better! Thanks for these improvements!

**Review by amenity — APPROVED** [review]
Oops! Forgot to actually "approve".

**Review by mateoclarke — APPROVED** [review]
map_anime

**Pull request merged**